yarn.lock

```diff
@@ -2600,7 +2600,7 @@
     react-transition-group "^4.4.2"
 
 "@mui/monorepo@https://github.com/mui/material-ui.git#master":
-  version "5.6.3"
+  version "5.6.4"
```

Comment:
yarn did id automatically, I guess we didn't update this correctly: https://github.com/mui/material-ui/blob/a09acabfc4affd241b227ebb9418c271fba873b8/package.json#L2